lib/character/codes.d.ts

```diff
@@ -5,60 +5,28 @@ export type Code = null | number
 // @for-script: REMOVE_ALL_THING_BELOW
 
 export interface Codes {
```

Comment:
Is there a way to not reorder this ?

Reply:
Looks like named exports are read in alphabetical order

Reply:
Hmm. While I get alphabetical, it makes the file rather unreadable. Would be preferable if we could find something for it

Reply:
Probably by adding a sort here? https://github.com/micromark/micromark/blob/b73fa448123f4c4b261318caa434f39c1195decf/script/generate-constant-typings.mjs#L11

Reply:
Fixed by using default export. In object keys order is preserved.